pyanaconda/modules/security/security.py

```diff
@@ -212,9 +221,18 @@ def collect_requirements(self):
         """
         requirements = []
 
+        # Add FIPS requirements.
+        if self.fips_enabled:
+            requirements.append(Requirement.for_package(
+                "/usr/bin/fips-mode-setup",
+                reason="Required for compliance."
```

Comment:
I recommend changing this to "Required for FIPS compliance", as there are more types of compliance than just compliance with FIPS.

Reply:
Fixed.